code/modules/atmospherics/gasmixtures/gas_mixture.dm

```diff
@@ -382,12 +419,11 @@ GLOBAL_LIST_INIT(gaslist_cache, init_gaslist_cache())
 		else
 			delta = delta * sharer_coeff
 
-		if(abs_temperature_delta > MINIMUM_TEMPERATURE_DELTA_TO_CONSIDER)
-			var/gas_heat_capacity = delta * gas[GAS_META][META_GAS_SPECIFIC_HEAT]
-			if(delta > 0)
-				heat_capacity_self_to_sharer += gas_heat_capacity
-			else
-				heat_capacity_sharer_to_self -= gas_heat_capacity //subtract here instead of adding the absolute value because we know that delta is negative.
+		var/gas_heat_capacity = delta * gas[GAS_META][META_GAS_SPECIFIC_HEAT]
+		if(delta > 0)
+			heat_capacity_self_to_sharer += gas_heat_capacity
+		else
+			heat_capacity_sharer_to_self -= gas_heat_capacity //subtract here instead of adding the absolute value because we know that delta is negative.
```

Comment:
I should probably just use a portion of the turfs heat capacity instead of doing this. I'll look into it later.